core/workspace_svg.js

```diff
@@ -1636,6 +1636,7 @@ Blockly.WorkspaceSvg.prototype.isDraggable = function() {
  * @package
  */
 Blockly.WorkspaceSvg.prototype.isContentBounded = function() {
```

Comment:
I worry that this function should have been updated, but it's purpose/usage is unclear to me.

Reply:
I just realized it's package, so it's only used internally. Therefore, I should remove it.